src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.test.tsx

```diff
@@ -3,17 +3,30 @@ import { render } from '#app/components/react-testing-library-with-providers';
 import PortraitVideoPromo from '.';
 
 describe('PortraitVideoPromo', () => {
-  it('Should contain a h3 level title', () => {
+  it('Should contain a title', () => {
     const sampleHeadlines = { promoHeadline: 'Sample Heading' };
 
     const { container } = render(
       <PortraitVideoPromo id="testId" headlines={sampleHeadlines} />,
     );
 
-    const heading = container.querySelector('h3');
+    const heading = container.querySelector('p')?.querySelector('span');
     expect(heading?.innerHTML).toBe('Sample Heading');
   });
 
+  it('Should contain a visually hidden text with required screen reader details for the component', () => {
+    const sampleHeadlines = { promoHeadline: 'Sample Heading' };
+
+    const { container } = render(
+      <PortraitVideoPromo id="testId" headlines={sampleHeadlines} />,
+    );
+
+    const heading = container.querySelector('strong');
+    expect(heading?.innerHTML).toBe(
+      'Sample Heading, video, undefined, Play video',
```

Comment:
Can we update the fixture data so that `undefined` is not read out?  But we should also update the code to ensure an undefined can never happen.
```suggestion
      'Sample Heading, video, undefined, Play video',
```